examples/clerk-nextjs/.env.example

```diff
@@ -0,0 +1,5 @@
+# Refer to the README on how to fill these variables
```

Comment:
I would prefix by framework (nextjs-clerk). That way they are grouped together by framework.